test/models/user_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  def setup
+  	@user = users(:one)
+  end	
+  
+  test 'user has many friends' do
+    assert 1 < @user.friends.count 
```

Comment:
Please run me through how you generated your friendships for your test user. I looked into your users fixture and didn't find any line there that you created a friendship.

Reply:
It's in the friendship fixtures, with associations with users we need